pages/views/components/esl-alert.njk

```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: ESL Alert
```

Comment:
As I understand it, the result will be:

ESL | ESL Alert

This is a very bad title! Unintelligible not only to the search engine but also to the potential visitor to the site. Why would he go to such a page?

The title should be something like this:

ESL Alert - a web component to show notifications on your pages

This is just a proposal.